main.js

```diff
@@ -124,3 +124,54 @@ exports.webView = async function (
         }, {})
     };
 };
+
+// overwrite default config to match OpenIDConnect authentication
+exports.config = async function (serv, serviceRole, serviceSession, config) {
+    return config ? true : {
```

Comment:
Could it be more convenient to have always an object, with a property in case the configuration is already ready ?
I don't like to have a different kind of result depending of the context. And it would allow easier evolution than a boolean.

Reply:
The idea was that partner may want to validate/process config when we set it and return `true` if all required options were set. Apparently meaning of the `config` parameter is far from obvious, so I'd better create a separate method to handle config validation.